.github/CODE_OF_CONDUCT.md

```diff
@@ -3,20 +3,20 @@
 This project adheres to the [eBay Code of Conduct](http://ebay.github.io/codeofconduct).
 By participating in this project you agree to abide by its terms.
 
-* Be friendly and patient.
```

Comment:
@mlrawlings I'm not sure if this was formatted this way for any reason, but prettier formatted it.

Reply:
Maybe I'm just seeing this is multiple projects, but it seems like prettier keeps switching between `-` and `*` for lists.  And it's driving me crazy.